sdk/trace/batch_span_processor.go

```diff
@@ -134,6 +140,7 @@ func (bsp *batchSpanProcessor) Shutdown(ctx context.Context) error {
 					otel.Handle(err)
 				}
 			}
+			bsp.isShutDown.Store(true)
```

Comment:
I think this can be handled without creating a new atomic member variable by setting `bsp.e = nil` once the `Exporter` is shutdown.  A short-circuit on `bsp.e == nil` should be added in `bsp.exportSpans()` if taking that approach and this method may need to take a lock on `bsp.batchMutex` around the modification of `bsp.e`.